File: app/src/main/java/com/github/ihsg/demo/ui/def/DefaultStyleActivity.java
package com.github.ihsg.demo.ui.def;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.github.ihsg.demo.R;


public class DefaultStyleActivity extends AppCompatActivity {

    public static void startAction(Context context) {
        Intent intent = new Intent(context, DefaultStyleActivity.class);
        context.startActivity(intent);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_default_style);

        findViewById(R.id.btn_setting).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                DefaultPatternSettingActivity.startAction(DefaultStyleActivity.this);
            }
        });

        findViewById(R.id.btn_checking).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                DefaultPatternCheckingActivity.startAction(DefaultStyleActivity.this);
            }
        });
    }
}

File: app/src/main/java/com/github/ihsg/demo/ui/def/DefaultStyleActivity.kt
package com.github.ihsg.demo.ui.def

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.github.ihsg.demo.databinding.ActivityDefaultStyleBinding


class DefaultStyleActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        ActivityDefaultStyleBinding.inflate(layoutInflater).apply {
            setContentView(root)

            btnSetting.setOnClickListener {
                DefaultPatternSettingActivity.startAction(this@DefaultStyleActivity)
            }
            btnChecking.setOnClickListener {
                DefaultPatternCheckingActivity.startAction(this@DefaultStyleActivity)
            }
        }
    }

    companion object {
        fun startAction(context: Context) {
            val intent = Intent(context, DefaultStyleActivity::class.java)
            context.startActivity(intent)
        }
    }
}
